website/content/docs/api-gateway/upgrades.mdx

```diff
@@ -101,6 +101,8 @@ If you are unable to tolerate any downtime, you can complete the following steps
 
 1. Change any existing `Gateways` to reference the new `GatewayClass` `consul`. Refer to [gatewayClass](/consul/docs/api-gateway/configuration/gateway#gatewayclassname) for additional information.
 
+1. After updating all of your `gateway` configurations to use the new controller, you can complete the upgrade again and completely remove the `apiGateway` block to remove the old controller. 
```

Comment:
```suggestion
1. After updating all of your `gateway` configurations to use the new controller, you can remove the `apiGateway` block from the Helm chart and rerun it. This will completely remove the old gateway controller. 
```
@t-eckert , can you confirm my proposed wording is correct.

Reply:
Yes, you remove the apiGateway stanza. 

It may be good to call out that this is the root apiGateway stanza as the configuration moved to the connectInject stanza.